Demos/SwiftUI Demo/SwiftUI Demo/ContentView.swift

```diff
@@ -13,13 +13,14 @@ struct ContentView: View {
     @State var isAuthenticated: Bool
     @State var authError: Error?
 
+    let resource = "https://api.logto.io"
     let client: LogtoClient?
 
     init() {
         guard let config = try? LogtoConfig(
-            endpoint: "https://logto.dev",
+            endpoint: "http://localhost:3001",
```

Comment:
This is for local test purpose, right? Should revert

Reply:
yes